docs: add instructions to disable venv prompt (python)

icy-comet opened the pull request:
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes/features)

### Description

Oh My Posh docs only contain directions to disable the Conda automatic prompt. This PR introduces directions to disable the automatic `venv` prompt change as well for the Python users.

<!---

Tips:

If you're not comfortable with working with Git, we're working a guide (https://ohmyposh.dev/docs/contributing_git) to help you out.
Oh My Posh advises GitKraken (https://www.gitkraken.com/invite/nQmDPR9D) as your preferred cross platform Git GUI power tool.

-->

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

icy-comet committed:
docs: add instructions to disable venv prompt (python)

Add details for Python users to disable the automatic prompt change with the `venv` library's virtual environments to solely rely on Oh My Posh for the info.

JanDeDobbeleer commented on docs/docs/faq.mdx:
````diff
@@ -138,6 +138,15 @@ To solely rely on Oh My Posh to set the prompt, you can configure the follwing s
+To disable this behaviour, set `VIRTUAL_ENV_DISABLE_PROMPT` environment variable to `1` on your system.
+You can add the following line to your Powershell profile:
+
+```powershell
````
Do we have guides for other shells too? I know it's only changing the syntax, but could be handy?

icy-comet replied:
just updated with a new commit @JanDeDobbeleer

icy-comet committed:
add export statements for a few more shells

Include directions and probable file locations to put the commands in for other shells.

icy-comet committed:
docs: define tilde char. in "venv prompt" section
